packages/components/src/Tooltip/Tooltip.mdx

```diff
@@ -32,6 +32,19 @@ import { Tooltip } from "@jobber/components/Tooltip";
   </Tooltip>
 </Playground>
 
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Tooltip component, consider the following:
+
+### Usage
+- A tooltip should be used to introduce a novel UI element or highlight a change to an existing interface.
+- A tooltip should contain information that is helpful, but not necessary for the user to successfully interact with the page.
+- A tooltip should not be a permanent fixture in the interface - if you need a message that remains in the view, consider putting that content inline. 
```

Comment:
This last one I'm unclear on. What do you mean a permanent fixture? A lot of tooltips are permanent in the interface

Reply:
I think that's more my ideal state going forward and less a reflection of current state... If something is hidden in a tooltip and we're just using that to save space it's probably not a great UI solution?

Reply:
I think there may always be supplementary information that are just not necessarily a problem of the UI (see: pricing grid and its million tooltips). Sure there may be other ways to handle that, but I do think tooltips have their place with permanence in a UI

Reply:
Yeah that's a fair point

There's also the idea of a tooltip as a bonus button label (which we don't do but could be kind of nice in the future) that systems like Material do pretty well

I think maybe the previous line about "helpful, but not necessary" covers what I was trying to say here